tests/models/mistral/test_modeling_mistral.py

```diff
@@ -462,39 +462,134 @@ def test_flash_attn_2_inference_padding_right(self):
+        model = MistralForCausalLM.from_pretrained(
+            "mistralai/Mistral-7B-v0.1", device_map="auto", torch_dtype=torch.float16
+        )
         input_ids = torch.tensor([input_ids]).to(model.model.embed_tokens.weight.device)
         with torch.no_grad():
             out = model(input_ids).logits.cpu()
         # Expected mean on dim = -1
         EXPECTED_MEAN = torch.tensor([[-2.5548, -2.5737, -3.0600, -2.5906, -2.8478, -2.8118, -2.9325, -2.7694]])
         torch.testing.assert_close(out.mean(-1), EXPECTED_MEAN, atol=1e-2, rtol=1e-2)
-        # slicing logits[0, 0, 0:30]
-        EXPECTED_SLICE = torch.tensor([-5.8781, -5.8616, -0.1052, -4.7200, -5.8781, -5.8774, -5.8773, -5.8777, -5.8781, -5.8780, -5.8781, -5.8779, -1.0787,  1.7583, -5.8779, -5.8780, -5.8783, -5.8778, -5.8776, -5.8781, -5.8784, -5.8778, -5.8778, -5.8777, -5.8779, -5.8778, -5.8776, -5.8780, -5.8779, -5.8781])  # fmt: skip
+
+        EXPECTED_SLICE = {
+            7: torch.tensor(
+                [
+                    -5.8781,
+                    -5.8616,
+                    -0.1052,
+                    -4.7200,
+                    -5.8781,
+                    -5.8774,
+                    -5.8773,
+                    -5.8777,
+                    -5.8781,
+                    -5.8780,
+                    -5.8781,
+                    -5.8779,
+                    -1.0787,
+                    1.7583,
+                    -5.8779,
+                    -5.8780,
+                    -5.8783,
+                    -5.8778,
+                    -5.8776,
+                    -5.8781,
+                    -5.8784,
+                    -5.8778,
+                    -5.8778,
+                    -5.8777,
+                    -5.8779,
+                    -5.8778,
+                    -5.8776,
+                    -5.8780,
+                    -5.8779,
+                    -5.8781,
+                ]
+            ),  # fmt: skip
+            8: torch.tensor(
+                [
+                    -5.8711,
+                    -5.8555,
+                    -0.1050,
+                    -4.7148,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -1.0781,
+                    1.7568,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                    -5.8711,
+                ]
+            ),  # fmt: skip
+        }
```

Comment:
```suggestion
        } # fmt: skip
```
should go here I think

Reply:
OK, I will remove the previous 2 and keep the last one.